sql/rename.go

```diff
@@ -68,3 +68,71 @@ func (p *planner) RenameDatabase(n *parser.RenameDatabase) (planNode, error) {
 
 	return &valuesNode{}, nil
 }
+
+// RenameTable renames the table.
+// Privileges: WRITE on database.
+//   Notes: postgres requires superuser, db owner, "CREATEDB", or table owner.
```

Comment:
WRITE on database is fine for now.
please update the comment to:
postgres requires the table owner.
mysql requires `ALTER, DROP` on the original table, and `CREATE, INSERT` on the new table (and does not copy privileges over)